controllers/openstackdataplanenodeset_controller.go

```diff
@@ -382,7 +382,7 @@ func (r *OpenStackDataPlaneNodeSetReconciler) SetupWithManager(mgr ctrl.Manager)
 
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&dataplanev1.OpenStackDataPlaneNodeSet{}).
-		Owns(&v1alpha1.OpenStackAnsibleEE{}).
+		Owns(&v1beta1.OpenStackAnsibleEE{}).
```

Comment:
I would not use v1beta1 here. How did this work when I tried the same thing with https://github.com/openstack-k8s-operators/dataplane-operator/pull/571/files and it did not work.